pkg/pipeline/bean/CiBuildConfig.go

```diff
@@ -28,15 +28,24 @@ type CiBuildConfigBean struct {
 }
 
 type DockerBuildConfig struct {
-	DockerfilePath     string            `json:"dockerfileRelativePath,omitempty"`
-	DockerfileContent  string            `json:"dockerfileContent"`
-	Args               map[string]string `json:"args,omitempty"`
-	TargetPlatform     string            `json:"targetPlatform,omitempty"`
-	Language           string            `json:"language,omitempty"`
-	LanguageFramework  string            `json:"languageFramework,omitempty"`
-	DockerBuildOptions map[string]string `json:"dockerBuildOptions,omitempty"`
-	BuildContext       string            `json:"buildContext,omitempty"`
-	UseBuildx          bool              `json:"useBuildx"`
+	DockerfilePath         string                   `json:"dockerfileRelativePath,omitempty"`
+	DockerfileContent      string                   `json:"dockerfileContent"`
+	Args                   map[string]string        `json:"args,omitempty"`
+	TargetPlatform         string                   `json:"targetPlatform,omitempty"`
+	Language               string                   `json:"language,omitempty"`
+	LanguageFramework      string                   `json:"languageFramework,omitempty"`
+	DockerBuildOptions     map[string]string        `json:"dockerBuildOptions,omitempty"`
+	BuildContext           string                   `json:"buildContext,omitempty"`
+	UseBuildx              bool                     `json:"useBuildx"`
+	UseBuildxK8sDriver     bool                     `json:"useBuildxK8sDriver,omitempty"`
```

Comment:
are we using k8s driver when no ``UseBuildxK8sDriver`` is passed ??

Reply:
no